src/main/java/uk/ac/cam/cl/dtg/isaac/dto/IsaacQuizDTO.java

```diff
@@ -73,4 +85,50 @@ public boolean getVisibleToStudents() {
 	public void setVisibleToStudents(boolean visibleToStudents) {
 		this.visibleToStudents = visibleToStudents;
 	}
+
+	@Nullable
+	public QuizFeedbackMode getDefaultFeedbackMode() {
+		return defaultFeedbackMode;
+	}
+
+	public void setDefaultFeedbackMode(QuizFeedbackMode defaultFeedbackMode) {
+		this.defaultFeedbackMode = defaultFeedbackMode;
+	}
+
+	@Nullable
+    public Integer getTotal() {
+        return total;
+    }
```

Comment:
There seem to be a lot of spacing inconsistencies in some of these files. I guess it's tabs/spaces?
We always use four spaces for indents, and GitHub uses tab ~ 6 spaces I think.
It might be too late to address this now, beyond a find and replace at the very end. I don't want to mess the diffs up with lots of whitespace changes.

Reply:
I'll do it on a separate commit.